src/Commands/SystemCommands.py

```diff
@@ -122,7 +122,3 @@ def systemCommands(cmd, uvm):
         if len(uvm.splitTuple.widgets) > 1:
             uvm.splitTuple.widgets.pop() # doesn't work for mix of split and vsplit
 
-    else:
```

Comment:
I think we need the bool so it doesn't run a site command with a similar name

Reply:
Either add back bool, or change systemCommandHandler to not use bool and redditCommands to not have duplicate command (maybe` if == or ==`sorta thing instead of `if in`). Either way works for me, this was just the path of least resistance.

Reply:
This case is handled in `commandHandlerClass.py` with my proposed changes: 

```python
if cmd.split()[0] not in (SystemCommandList + ChanCommandList + RedditCommandList):
            return
        elif cmd.split()[0] in SystemCommandList:
            systemCommands(cmd, self.uvm)
        elif self.uvm.currFocusView.site is SITE.FCHAN and cmd.split()[0] in ChanCommandList:
            chanCommands(cmd, self.uvm)
        elif self.uvm.currFocusView.site is SITE.REDDIT and cmd.split()[0] in RedditCommandList:
            redditCommands(cmd, self.uvm)

        self.postCommand()
```

If a user were to enter a command, 's', then only the systemCommands() func would be called. is that desired behavior?

Reply:
oic yea that's fine